src/cli/commands/common/prompt-for-aws-target.ts

```diff
@@ -126,3 +155,138 @@ function toClusterChoice(dbCluster: AwsDbCluster): DistinctChoice {
     },
   };
 }
+function toElasticacheClusterChoices(
+  clusters: awsElasticacheCluster[][],
+  commandType: string,
+  elasticacheNodes: awsElasticacheCluster[]
+): DistinctChoice[] {
+  if (clusters.length === 0) {
+    return [];
+  }
+  return commandType === 'init'
+    ? toInitElasticacheClusterChoices(clusters)
+    : [
+        new inquirer.Separator('Redis clusters:'),
+        ...toConnectElasticacheClusterChoices(clusters, elasticacheNodes),
+      ];
+}
+
+function toInitElasticacheClusterChoices(
+  clusters: awsElasticacheCluster[][]
+): Array<DistinctChoice<ElasticacheClusterTargetInput>> {
+  return [
+    new inquirer.Separator('Elasticache clusters:'),
```

Comment:
The separators are different here and on line 169. Let's remove the duplication entirely by having it added once on the ternary operator level